static/css/index.css

```diff
@@ -1,85 +1,69 @@
 
-#index-banner {
-	background: url("../img/layout/openttd-index.png") repeat-x;
-	height: auto !important;
-	height: 183px;
+.index-banner {
+	height: 185px;
+	overflow: hidden;
+	border-radius: 6px;
```

Comment:
oeh, found more nitpicks: please use your abc for CSS :) (goes for more entries below)